packages/jaeger-ui/test/jest-per-test-setup.js

```diff
@@ -26,7 +26,25 @@ const Enzyme = require('enzyme');
 const EnzymeAdapter = require('@wojtekmaj/enzyme-adapter-react-17');
 const createSerializer = require('enzyme-to-json').createSerializer;
 
-Enzyme.configure({ adapter: new EnzymeAdapter() });
+// Suppress React 18 warnings about ReactDOM.render in tests
```

Comment:
warnings are there for a reason. I know it's annoying but they typically not interfering with understanding test failures.